test/drivers/second_quantization/gaussiand/test_driver_methods_gaussian.py

```diff
@@ -10,18 +10,18 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-""" Test Driver Methods Gaussian """
+""" Test Driver Problem Gaussian """
 
 import unittest
 
-from test.drivers.second_quantization.test_driver_methods_gsc import TestDriverMethods
+from test.drivers.second_quantization.test_driver_problem import TestDriverProblem
 from qiskit_nature.drivers.second_quantization import GaussianDriver
 from qiskit_nature import QiskitNatureError
 from qiskit_nature.transformers.second_quantization import FreezeCoreTransformer
 
 
-class TestDriverMethodsGaussian(TestDriverMethods):
-    """Driver Methods Gaussian tests"""
+class TestDriverProblemGaussian(TestDriverProblem):
```

Comment:
The original rationale why this was called TestDriverMethods was that it was really around tests for the different methods - rhf, rohf, uhf

Reply:
I see. I renamed it because these tests are now all based on the `Problem` classes whereas the other drivers tests are more basic. I don't think it matters too much though.

Reply:
They were named based on _what_ they were testing not _how_ - i.e. it's not about the problem, that's just a means to an end to verify the function/result. If we could exact diagonalize a FermionicOp to get a min eig value it could be done that way given what is tested perhaps - before we used to use core Hamiltonian and Numpy - using Problem and GroundStateSolver is just the latest way.

Reply:
After an offline-sync and re-evaluating my view on these tests I have decided to revert this naming change. Initially I was thinking differently about this but I agree that this is a change which does not add any value. I will revert this later today.